controllers/storageclaim_controller.go

```diff
@@ -382,6 +382,9 @@ func (r *StorageClaimReconciler) reconcilePhases() (reconcile.Result, error) {
 					csiClusterConfigEntry.CephFS.SubvolumeGroup = data["subvolumegroupname"]
 					// delete groupname from data as its not required in storageclass
 					delete(data, "subvolumegroupname")
+					csiClusterConfigEntry.CephFS.KernelMountOptions = data["kernelmountoptions"]
```

Comment:
@rohan47 can you please define a const for `kernelmountoptions` and please link the ocs-operator PR to give context there is already PR which is doing this change in other places.

Reply:
Made the change, Updated PR description with ocs-operator PR link